web/themes/contrib/civictheme/includes/cards.inc

```diff
@@ -172,6 +172,17 @@ function civictheme_preprocess_paragraph__civictheme_subject_card(array &$variab
   _civictheme_preprocess_paragraph__paragraph_field__theme($variables);
 }
 
+/**
+ * Implements template_preprocess_node().
+ */
+function _civictheme_preprocess_node__civictheme_subject_card(array &$variables): void {
+  _civictheme_preprocess_paragraph__node_field__image($variables, CivicthemeConstants::SUBJECT_CARD_IMAGE_STYLE);
```

Comment:
We test for render of `civictheme_subject_card` here:
`tests/behat/features/paragraph.civictheme_subject_card_ref.render.feature`

How was the test passing in the past without this preprocess?

Reply:
The test is for paragraph render when used with manual list 
which uses either 
civictheme_preprocess_paragraph__civictheme_subject_card
or 
civictheme_preprocess_paragraph__civictheme_subject_card_ref

but what we added now is more of a feature. Which we dont allow by default in civic theme 
we only had 
for 
_civictheme_preprocess_node__civictheme_navigation_card
_civictheme_preprocess_node__civictheme_promo_card
_civictheme_preprocess_node__civictheme_snippet
![Screenshot 2024-09-19 at 10 42 31 AM](https://github.com/user-attachments/assets/43d6ad58-370c-4979-867b-7d44f88f955f)

Reply:
Happy with that if we do add it as a feature we can then add tests for it.